### PR TITLE
feat(breakdowns): Add span_op_breakdowns fields to transactions and discover datasets.

### PR DESCRIPTION
**Pull request by dashed**
Depends on https://github.com/getsentry/snuba/pull/1762

Associated change in Relay: https://github.com/getsentry/relay/pull/934

Add `span_op_breakdowns` fields to the transaction and discover datasets so they can be read from Sentry.

## TODO

- [x] add tests for aggregate usage

### CODE REVIEW

File: tests/test_discover_api.py [hunk 1]
```diff
@@ -866,6 +866,54 @@ def test_individual_measurement(self) -> None:
...
+        data = json.loads(response.data)
+        assert response.status_code == 200, response.data
+        assert len(data["data"]) == 1, data
+        assert data["data"][0]["event_id"] != "0"
+        assert data["data"][0]["span_op_breakdowns[ops.db]"] == 62.512
+        assert data["data"][0]["span_op_breakdowns[ops.http]"] == 109.774
+        assert data["data"][0]["span_op_breakdowns[total.time]"] == 172.286
+        assert data["data"][0]["span_op_breakdowns[not_found]"] is None
+
+        response = self.post(
```
Comment (evanh) [inline tests/test_discover_api.py:898]: Why is this here?

File: tests/test_discover_api.py [hunk 2]
```diff
@@ -866,6 +866,54 @@ def test_individual_measurement(self) -> None:
         assert "measurements[lcp]" in data["data"][0]
         assert data["data"][0]["measurements[lcp]"] is None
 
+    def test_individual_breakdown(self) -> None:
```
Comment (evanh) [inline tests/test_discover_api.py:869]: Having a test that closely mirrors how you intend to use this feature would also be nice. E.g. if you plan to aggregate/use them in conditions.

File: tests/test_discover_api.py (hunk 1 above)
Comment (dashed) [reply tests/test_discover_api.py:898]: Copied it from existing test `test_individual_measurement`. Forgot to rename things to span_op_breakdowns

File: tests/test_discover_api.py [hunk 3]
```diff
@@ -1004,6 +1052,97 @@ def test_web_vitals_histogram_function(self) -> None:
         assert response.status_code == 200, response.data
         assert len(data["data"]) == 0, data
 
+    def test_span_op_breakdown_histogram_function(self) -> None:
+        response = self.post(
+            json.dumps(
```
Comment (dashed) [inline tests/test_discover_api.py:1057]: This query was generated from a real Snuba query from my testing ingesting events (end to end) locally and producing aggregated/histogram values on the product side (i.e. frontend).

**Review by evanh — APPROVED** [review]
Please do not merge until the migration has been successfully run.

**Comment by dashed** [comment]
Will merge this to master after https://github.com/getsentry/snuba/pull/1762 is merged

**Pull request merged**